crates/bevy_render/src/render_resource/buffer_vec.rs

```diff
@@ -28,7 +34,7 @@ use wgpu::BufferUsages;
 /// * [`GpuArrayBuffer`](crate::render_resource::GpuArrayBuffer)
 /// * [`BufferVec`]
 /// * [`Texture`](crate::render_resource::Texture)
-pub struct BufferVec<T: Pod> {
+pub struct RawBufferVec<T: Pod> {
```

Comment:
I can't add a suggestion but the doc comment has a few mentions to `BufferVec` that should be renamed.